package.json

```diff
@@ -5,7 +5,7 @@
 , "repository": "goto-bus-stop/xkcd-37"
 , "author": "René Kooi <rene@kooi.me>"
 , "scripts":
-  { "test": "standard && node ./test.js" }
+  { "test": "standard index.js && node ./test.js" }
```

Comment:
I ran into a problem with standard here, during the test it threw an error for using a backslash randomly but was needed to complete the tests.